### PR TITLE
Changes in support of Issue #175

### PR DESCRIPTION
**Pull request by Orbisman**
Pull request for Issue #175..   

Dependent on Issue \#7 on the [MSV project](https://github.com/xmlark/msv/issues/7) being approved and code accepted.

New JUnit w/supporting files included.

Note: pom file change should not be included, my mistake.

### CODE REVIEW

**Comment by cowtowncoder** [comment]
Minor update to `pom.xml` wrt `msv-core` dependency; updated to latest non-snapshot in `master`.

File: pom.xml [hunk 1]
```diff
@@ -36,15 +37,23 @@ SAX2 and Stax2 APIs
...
+      <version.msv>2022.8-SNAPSHOT</version.msv>
       <version.plugin.javadoc>3.1.1</version.plugin.javadoc>
 
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>        
+      <skipTests>false</skipTests>
+        
       <!-- Woodstox 5.0 is 1.6+ -->
-      <javac.src.version>1.6</javac.src.version>
-      <javac.target.version>1.6</javac.target.version>
+      <jdk.version>1.8</jdk.version>
```
Comment (cowtowncoder) [inline pom.xml:50]: Will then need to go in after #134 is done (which itself is after 6.6.0 release). Probably not a problem as MSV-core change may take a while.

**Comment by Orbisman** [comment]
That works, just wanted to finish up the code and get it uploaded before I
forgot..

**Comment by cowtowncoder** [comment]
@Orbisman Is this still active? While the referenced issue in MSV was closed, there doesn't appear to be a new build available so I don't think this can be completed... but maybe at least could trim down changes to `pom.xml`?

**Comment by Orbisman** [comment]
Sorry for the delayed response, yes, remove the POM changes that was in error...

**Comment by Orbisman** [comment]
Going to resubmit pull request without pom

**Pull request closed without merging**